src/examples/zkapps/dex/erc20.ts

```diff
@@ -0,0 +1,211 @@
+import {
```

Comment:
draft of an ERC20-like token interface, not used yet but maybe will use later.
not everything ERC20 has is easily possible on Mina though (or possible at all, not sure)